plugin/hpsa/hpsa.py

```diff
@@ -74,38 +74,38 @@ def _sys_status_of(hp_ctrl_status):
 def _parse_hpssacli_output(output):
     """
     Got a output string of hpssacli to dictionary(nested).
-    Skipped these line:
-        1. Starts with 'Note:'
-           This is just a message right after controller. We don't neet it
-           yet.
-        2. The 'Physical Drives' line.
-           It should indented after 'Internal Drive Cage' like.
-           If not ignored, we might got duplication line error.
-           After ignored, it's phsycial disks will directly stored as
-           key of 'Internal Drive Cage' dictionary.
+    Workflow:
+    0. Check out top and the second indention level's space count.
+    1. Check current line and next line to determine whether current line is
+       a start of new section.
+    2. Skip all un-required sections and their data and sub-sections.
```

Comment:
So, essentially, there's a whitelist of required_sections that breaks your rules about indents? I think that's going to miss a lot of valuable data. As long as this is working for you know, I'm ok with it, but I'll want to writeup an issue to document what this new logic is intentionally not collecting. Make sense?